File: C Programming/C Calculator/calc.c
#include <stdio.h>

int main()
{
	int num1, num2, result, instances;
	char operation;
	printf("Input the number of instances: \n");
	scanf("%d", &instances);
	for (int i = 0; i < instances; ++i)
	{
		printf("Select an operation: \n");
		scanf("%c", &operation);
		switch (operation) {
			case ('+'):
				printf("Input two numbers: \n");
				scanf("%d", &num1);
				scanf("%d", &num2);
				result = num1 + num2;
				printf("The sum is %d\n", result);
				break;
			case ('-'):
				printf("Input two numbers: \n");
				scanf("%d", &num1);
				scanf("%d", &num2);
				result = num1 - num2;
				printf("The difference is %d\n", result);
				break;
			case ('*'):
				printf("Input two numbers: \n");
				scanf("%d", &num1);
				scanf("%d", &num2);
				result = num1 * num2;
				printf("The product is %d\n", result);
				break;
			case ('/'):
				printf("Input two numbers: \n");
				scanf("%d", &num1);
				scanf("%d", &num2);
				result = num1 / num2;
				printf("The quotient is %d\n", result);
				break;
			default:
				break;
			}
	}
	return 0;
}

File: Python Programming/PyCalc/calc.py
def add(x, y):
    result = x + y
    print(f'The sum is {result}.')


def subtract(x, y):
    result = x - y
    print(f'The difference is {result}.')


def multiply(x, y):
    result = x * y
    print(f'The product is {result}.')


def divide(x, y):
    result = x / y
    print(f'The quotient is {result}.')


print("Enter the number of instances: ")
instances = int(input())

while instances > 0:
    instances -= 1
    print("Select an operation: ")
    operation = input()
    if operation == "add":
        print("Input two numbers: ")
        num1 = int(input())
        num2 = int(input())
        sum = add(num1, num2)
        print(f"The sum is {sum}.")
    elif operation == "subtract":
        print("Input two numbers: ")
        num1 = int(input())
        num2 = int(input())
        difference = subtract(num1, num2)
        print(f"The difference is {difference}.")
    elif operation == "multiply":
        print("Input two numbers: ")
        num1 = int(input())
        num2 = int(input())
        product = multiply(num1, num2)
        print(f"The product is {product}.")
    elif operation == "divide":
        print("Input two numbers: ")
        num1 = int(input())
        num2 = int(input())
        quotient = divide(num1, num2)
        print(f"The quotient is {quotient}.")
    else:
        print("Invalid operation!")
